src/pages/preferences/preferencesTabs/cliTab/AuthenticationCard.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Button, { Variant } from "@leafygreen-ui/button";
+import { Subtitle } from "@leafygreen-ui/typography";
+import Styled from "@emotion/styled";
+import Code from "@leafygreen-ui/code";
+import { SiderCard } from "components/styles";
+
+export const AuthenticationCard = () => {
+  const authCode = `
```

Comment:
Is there a ticket to make a request for this data?

Reply:
https://jira.mongodb.org/browse/EVG-8066